content/get-started/cli-cmd-wallet-multisig-signature-create.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: wallet:multisig:signature:create
+seo: Wallet Multisig Signature Create command
+---
+
+# Usage
+
+Creates a signature share for a participant for a given signing package. The signing package contains an unsigned transaction and signing commitments.
```

Comment:
```suggestion
Creates a signature share for a participant for a given signing package.
```

It is not obvious to the end user that the signing package contains these two things. Should we just exclude this wording?

Reply:
I don't feel strongly about keeping it in. I added it as a way to explain because the end user might not know what a signing package is